spec/services/coronavirus/pages/content_builder_spec.rb

```diff
@@ -61,13 +28,15 @@
     end
 
     before do
-      data["header_section"] = header_json
-      data["sections"] = [sub_section_json]
-      data["hidden_search_terms"] = hidden_search_terms
+      @data = {}
+      @data["title"] = "Coronavirus (COVID-19): guidance and support"
+      @data["header_section"] = header_json
+      @data["sections"] = [sub_section_json]
+      @data["hidden_search_terms"] = hidden_search_terms
     end
 
-    it "returns github and model data" do
-      expect(subject.data).to eq data
+    it "returns model data" do
+      expect(subject.data).to eq @data
```

Comment:
Would be really nice to keep `data` instead of creating `@data`. Maybe move the `before` block into the `it` test, then `data` becomes possible, and since it's only use for this test...?

Reply:
I like that suggestion. I've updated and fixed up.